src/vendor/processors/stripe.py

```diff
@@ -333,22 +332,50 @@ def get_stripe_connect_account(self):
 
         return None
 
+    def get_stripe_base_fee_amount(self, amount):
+        stripe_base_fee = 0
+
+        if 'percentage' in STRIPE_BASE_COMMISSION and STRIPE_BASE_COMMISSION['percentage']:
```

Comment:
totally not a blocker but easier way to check if key is both in the dict and has a value is `if STRIPE_BASE_COMMISSION.get("percentage"):`

same as the check below